tasks/code_sync_status.rb

```diff
@@ -0,0 +1,93 @@
+#!/opt/puppetlabs/puppet/bin/ruby
+
+# Puppet Task Name: code_sync_status
+require 'net/https'
+require 'uri'
+require 'json'
+require 'puppet'
+
+# CodeSyncStatus task class
+class CodeSyncStatus
+  def initialize(params)
+    @params = params
+  end
+
+  def execute!
+    puts syncstatus.to_json
+  end
+
+  def https
+    https = Net::HTTP.new('localhost', '8140')
```

Comment:
The same comment applies to this variable, should we change it to just `client`? since we are in the `https_client` context (method)

Reply:
Yes good shout